terraform/modules/prom-ec2/paas-config/prometheus.conf.tpl

```diff
@@ -69,3 +69,163 @@ scrape_configs:
     scheme: https
     static_configs:
       - targets: ['canary.verify-main.london.verify.govsvc.uk']
+  - job_name: paas_elasticsearch_for_dm
```

Comment:
I think I'd rather pass the whole scrape_config into the module as one big string, with a default of the empty string.  We'd then template and construct the scrape_config in `terraform/projects/prom-ec2/paas-production`.

Why?

 - so that our staging observe prometheus doesn't try to scrape this stuff
 - to keep explicit references to DM out of the supposedly-generic prom-ec2 module

Reply:
This is a good idea. I'll restructure the PR to do it. 👍